ci-operator/jobs/cri-o/test/cri-o-test-main-presubmits.yaml

```diff
@@ -0,0 +1,118 @@
+presubmits:
+  cri-o/test:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^main$
+    - ^main-
+    cluster: build01
+    context: ci/prow/example
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/cloud: equinix-ocp-metal
+      ci-operator.openshift.io/cloud-cluster-profile: equinix-ocp-metal
```

Comment:
where did this profile come from? is it predefined

Reply:
it is predefined :ok_hand: it's used in a lot of other places but I'm not sure where it's actually created